doc/kconfig/conf.py

```diff
@@ -59,7 +58,14 @@
 # If true, license is shown in the HTML footer. Default is True.
 html_show_license = True
 
+html_theme_options = {
+    "versions": {
+        "ncs": "nRF Connect SDK",
```

Comment:
This must be nrf. The link back from another doc set doesn't work.

Reply:
fixed